### PR TITLE
Enable local indexing on license master

### PR DESCRIPTION
**Pull request by nwang92**
#550

### CODE REVIEW

**Review by jrigassio-splunk — APPROVED** [review]

File: roles/splunk_common/tasks/enable_forwarding.yml [hunk 1]
```diff
@@ -51,22 +51,19 @@
...
   register: default_tcpout_group
 
 # NOTE: If this task is called or used, it will disable all local indexing!
 - name: Disable indexing on the current node
   ini_file:
     path: "{{ splunk.home }}/etc/system/local/outputs.conf"
     section: "indexAndForward"
-    option: "{{ item.key }}"
-    value: "{{ item.value }}"
+    option: "index"
```
Comment (bb03) [inline roles/splunk_common/tasks/enable_forwarding.yml:61]: Why did you change this from the withitems loop?  Because these options will never expand or change?

**Review by bb03 — APPROVED** [review]

File: roles/splunk_common/tasks/enable_forwarding.yml (hunk 1 above)
Comment (nwang92) [reply roles/splunk_common/tasks/enable_forwarding.yml:61]: Yeah, I figured we weren't really using the loop very much with just 1 item. The `indexAndForward` section only exposes 2 possible keys anyways: https://docs.splunk.com/Documentation/Splunk/8.0.6/Admin/Outputsconf#IndexAndForward_Processor-----

**Review by sarahotis — APPROVED** [review]
